pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+filterwarnings =
+    ignore:::lark.*
```

Comment:
Not needed?

Reply:
done in Done in [comments from Alec: clean up CI, pylint, mypy](https://github.com/CQCL/quantinuum-qircheck/pull/1/commits/7e7439ff6b495e25996a4423757e99735675f3ff)
[7e7439f](https://github.com/CQCL/quantinuum-qircheck/pull/1/commits/7e7439ff6b495e25996a4423757e99735675f3ff)